docker-compose.yaml

```diff
@@ -24,11 +24,15 @@ services:
 
   rest-api-model-server:
     image: modelix/rest-api-model-server:latest
+    # Needs to be specified, so that container runs in Docker on Mac with ARM
+    platform: linux/x86_64
     profiles:
-      - useCase2a
+      - useCase2b
     ports:
-      - 8090:8090
-      - 8080:8080
+      # The dashboard expects our service to be running under 8090.
+      - 8090:8080
```

Comment:
The default port was always 8080.
In our `application.properties` we set it to `8090` for the `dev` profile
See `%dev.quarkus.http.port=8090`